manifestinteractive/jqvmap
# Fixes Mouse Drag Map Release Issue

#### What does this PR do?

Fixes Mouse Drag Map Release Issue

#### How should this be manually tested?

1. Open any map in the `./examples` folder
2. Zoom the map to 3-4 levels.
3. Drag the map 
4. While dragging map move the cursor out of map area ( out of map frame) and release the cursor (mouseup)
5. Now just move the cursor back into map area, the map should not move as if it were still being dragged. ( previous version would still act like you were dragging ).

#### What are the relevant tickets?

Fixes #118 

#### What gif best describes how you feel about this work?

![thinking](https://cloud.githubusercontent.com/assets/508411/11616649/baabab76-9c4e-11e5-9d0c-42c73474e313.gif)
